consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/api/CMCApi.java

```diff
@@ -27,6 +31,12 @@ public interface CMCApi {
 
     String SDT_REQUEST_ID = "SDTREQUESTID";
 
+    String SERVICE_AUTHORIZATION = "ServiceAuthorization";
+
+    String FROM_DATE_TIME = "fromDateTime";
+
+    String TO_DATE = "toDate";
```

Comment:
Should this be called TO_DATE_TIME?  I believe the to date has a time component as well.